src/main/java/make_post/use_case/MakePostInteractor.java

```diff
@@ -12,9 +12,13 @@
 public class MakePostInteractor implements MakePostInputBoundary {
     private final MakePostDsGateway dataAccess;
     private final MakePostOutputBoundary presenter;
-
     private final PostFactory postFactory = new PostFactory();
```

Comment:
The only thing is that the post probably should be passed in the constructor, but thats not a huge issue and I can add this as I'm working on putting the different iterations of post and user factory together into one